requirements.txt

```diff
@@ -9,4 +9,5 @@ pydantic
 quimb>=1.4.0
 scipy
 sympy>=1.9
+pyzx==0.8.0
```

Comment:
Is it really necessary to pin the version?

Reply:
PyZX breaks very frequently, so I believe it is better to pin the version to prevent troubles in future

Reply:
@shinich1 

Is it really acceptable to include `pyzx` as a new dependency?

Reply:
I would keep `pyzx` optional (no inclusion in `requirements.txt` and try the import only when using `opengraph` class or equivalent.

Reply:
Sure

Reply:
If it isn't in the requirements file though, it will be a problem when the unit tests are run. I'm not sure it will even pass the CI pipeline.
I could include it in a different requirements file if that helps? Or explicitly install pyzx in the github workflow pipeline. Though this would be confusing for people who run `tox` without knowing about this

Reply:
There's actually a similar example with `rustworkx`. We include in `requirements-dev.txt` and ask [dev] installation for those contributing (see https://github.com/TeamGraphix/graphix/blob/master/CONTRIBUTING.md). It'll be fine if we add `pyzx` in `requirements-dev`.

@EarlMilktea I understand type annotations can be done similarly to `rustworkx` case, please correct my if I'm wrong about this.

Reply:
Sure. I've added it into the `requirements-dev.txt` file